app/utils.py

```diff
@@ -115,3 +115,12 @@ def last_n_days(limit_days):
         # reverse the countdown, -1 from first two args to ensure it stays 0-indexed
         for x in range(limit_days - 1, -1, -1)
     ]
+
+
+def escape_special_characters(string):
```

Comment:
It would be nice to use this function in app.dao.notifications_dao.dao_get_notifications_by_to_field as well.

Reply:
I think I did it already: https://github.com/alphagov/notifications-api/pull/1943/files#diff-e680b43002cba79b095e3ddaa3237536R455

Unless I should use it in a different place in that function, too?